source/includes/events/_bill.scheduled-payment-notice.md.erb

```diff
@@ -0,0 +1,43 @@
+
+
+## Bill scheduled payment notice
+
+> Example `bill.scheduled-payment-notice` Event
+
+```json
+{
+  "event": {
+    "_id": "580e3e11dec21e861b5a3719",
+    "type": "bill.scheduled-payment-notice",
+    "created": "2016-04-20T16:51:12Z",
+    "clientId": "579b695decfa110127118752",
+    "partnerId": "55942f330cfcfd6c4ed529d9",
+    "identifier": "example-id-123",
+    "payload": {
+      "resource": {
+        "type": "bill",
+        "id": "579b695decfa11012711875d"
+      },
+      "data": {
+        "company": {
+          "name": "XYZ Electric"
+        },
+        "bill": {
+          "dueDate": {
+            "previous": "2019-03-20 17:00:00.000",
+            "current": "2019-03-28 17:00:00.000"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Friendly notice for when the due date changes after a bill payment has been scheduled.
+
+Attribute | Description
+--------- | -----------
+`type` | `bill.scheduled-payment-notice`
+`payload.data.company` | The [company object](#the-company-object) linked to this resource.
+`payload.data.bill` | Object containing the names of the bill attributes that have changed and their previous values if available.
```

Comment:
The bill object containing attributes that have changed and their current and previous values, if available.

- I added a comma before if
- Based on the example in the docs, from what I can see the payload.data.bill returns the current and previous attributes (previous if available) from the bill object, so I rewrote it to better reflect that. I also rewrote to follow the same format as the payload.data.company above it -- which I would recommend linking "bill object" to the Bill object in the docs, like the company object.